docs/commands/bundle.md

```diff
@@ -73,7 +77,7 @@ redocly bundle --dereferenced --output dist --ext json openapi/openapi.yaml open
 JSON output only works when there are no circular references.
 {% /admonition %}
 
-### Custom configuration file
+### Use custom configuration file
```

Comment:
```suggestion
### Use alternative file path
```

Reply:
Updated per suggestion.

Reply:
Corrected to "filepath" to be consistent with the other command docs and [English convention](https://english.stackexchange.com/questions/428215/filepath-or-file-path).